pyproject.toml

```diff
@@ -7,7 +7,7 @@ authors = ["SatoshiLabs <info@satoshilabs.com>"]
 [tool.poetry.dependencies]
 python = "^3.11"
 termcolor = "^1.1.0"
-trezor = { git = "https://github.com/trezor/trezor-firmware.git", branch = "main", subdirectory = "python" }
+trezor = { git = "https://github.com/trezor/trezor-firmware.git", branch = "M1nd3r/thp", subdirectory = "python" }
```

Comment:
also this branch seems to be merged already. is THP trezorctl support finished in main? @matejcik